sdk/react-native/package.json

```diff
@@ -28,9 +28,11 @@
     "dependencies": {
         "@devcycle/js-client-sdk": "^1.32.2",
         "@devcycle/react-client-sdk": "^1.30.2",
+        "@devcycle/types": "^1.19.2",
         "@react-native-async-storage/async-storage": "^1.17.11",
         "react-native-device-info": "^8.7.0",
-        "react-native-get-random-values": "^1.7.2"
+        "react-native-get-random-values": "^1.7.2",
+        "react-native-sse": "^1.2.1"
```

Comment:
note: this will require docs update as we need to install the other packages separately

Reply:
Why wouldn't it be installed from the dependency? seems to work in the E2E test app.

Reply:
im not 100 percent sure, but we have that explicit in the docs and in the past when i have tried to install, required me to add those dependencies have not tested recently tbh